src/WebJobs.Extensions.DurableTask/LinuxLogger.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
+{
+    public class LinuxLogger : ILogger
```

Comment:
Since this is a platform-specific thing, I would suggest naming it `AppServiceLinuxLogger` or something similar.